tests/page-objects/messages/sms-gateway.po.js

```diff
@@ -0,0 +1,103 @@
+const utils = require('../../utils');
+const commonElements = require('../common/common.po');
+const messagePo = require('../messages/messages.po');
+const reportsPo = require('../reports/reports.po');
+const helper = require('../../helper');
+const { browser, element } = require('protractor');
+
+//page objects
+const getState = (first, second) => {
+  return element(by.css(
+    `#reports-content .scheduled-tasks > ul > li:nth-child(${first}) > ul > li:nth-child(${second}) .task-state .state`
+  ));
+};
+const  sentTask = element(by.css('#reports-content .details > ul .task-list .task-state .state'));
+
+const deliveredTask = getState(1,1);
+
+const scheduledTask = getState(1,2);
+
+const failedTask = getState(2,1);
```

Comment:
This all seems tied to the specific test case so if we try and use this PO for a new test case it'll probably fail, right? Can we make this more generic and reusable somehow?

Reply:
I see what you mean. It is updated.